_docs/concepts/handling-failures.md

```diff
@@ -0,0 +1,76 @@
+---
+category: Concepts
+title: Handling Failures
+
+parent: Traffic Management
+order: 30
+
+bodyclass: docs
+layout: docs
+type: markdown
+---
+
+Envoy sidecar/proxy provides a set of out-of-the-box _opt-in_ failure recovery features
+that can be taken advantage of by the services in an application. Features
+include:
+
+1. Timeouts
+2. Bounded retries with timeout budgets and variable jitter between retries
+3. Limits on number of concurrent connections and requests to upstream services
+4. Active health checks on each member of the load balancing pool
```

Comment:
Passive health checks is outlier detection (circuit breaker per instance in the LB pool). This is the 5th point. But I will clarify